lib/joke.rb

```diff
@@ -0,0 +1,8 @@
+class Joke
+  attr_reader :id, :setup, :punchline
+  def initialize(id, setup, punchline)
```

Comment:
Nice work with this class! Tiny thing-- ruby style guide prefers that you leave a space after your attr_reader line, before `def initialize`